3-cloud-computing/3.2.1-s3-cloudfront.md

```diff
@@ -6,26 +6,20 @@ In this section we will run through several exercises which showcase AWS S3 and
 
 > Amazon Simple Storage Service (Amazon S3) is the largest and most performant, secure, and feature-rich object storage service. With Amazon S3, organizations of all sizes and industries can store any amount of data for any use case, including applications, IoT, data lakes, analytics, backup and restore, archive, and disaster recovery. -- [Amazon S3](https://aws.amazon.com/s3/)
 
-For this exercise we're hosting a static HTML website using AWS S3. We will be using a simple react based application called [DevOps Knowledge Share](https://github.com/liatrio/devops-knowledge-share-dob-ui) to create our sample website. This small scale project is owned and managed here at Liatrio and its intended purpose is to show our clients during dojos the power of CI/CD. It is a simple application that has a frontend (ui) and backend (api) but, we will only be using the ui for now. Feel free to try creating and substituting with a different template or project.
+For this exercise we're hosting a static HTML website using AWS S3. We will be using a simple react based application called [S3 Realworld UI DOB](https://github.com/liatrio/s3-realworld-ui-dob) to create our sample website. Feel free to try creating and substituting with a different template or project.
 
 **Requirements**
 
 - You will need to have npm installed to perform this exercise.
 
 1. Clone the sample project.
-2. Change to the template directory.
-3. Generate the HTML for the template.
+2. Navigate to the project directory directory. Run `npm install` to fetch all project dependencies.
+3. Generate the HTML for the templates with `npm run build`
+4. Run `npm start` and open http://localhost:3000 to confirm that the website is set up correctly.
+5. Create the S3 Bucket. Replace *YOUR_NAME* with your name and *PROFILE_NAME* with the name of the AWS cli profile you set up. Make sure to replace *PROFILE_REGION* with the region you configured you AWS cli profile to talk to (run `cat ~/.aws/config` to see what your cli is configured to).
 
 ```
-npm install
-npm run build
-```
-
-4. **Optional** Run `npm start` and open http://localhost:4100 to confirm that the website is set up.
-5. Create the S3 Bucket. Replace *YOUR_NAME* with your name and *PROFILE_NAME* with the name of the AWS cli profile you set up. The region requires a location constraint to be specified because it is not in us-east-1.
-
-```
-aws-vault exec PROFILE_NAME -- aws s3api create-bucket --bucket YOUR_NAME-dob-react --create-bucket-configuration LocationConstraint=us-west-2
+aws-vault exec PROFILE_NAME -- aws s3api create-bucket --bucket YOUR_NAME-dob-react --create-bucket-configuration LocationConstraint=PROFILE_REGION.
```

Comment:
we could honestly add a section of learning what this string of stuff means. I had to pour over mans for days before I understood it.

Reply:
We could probably add a section to **1.3 The Basics** about man pages as I have found them very useful for exploring the aws cli